conans/model/conan_file.py

```diff
@@ -234,6 +234,10 @@ def package_folder(self):
     def package_folder(self, folder):
         self.folders.set_base_package(folder)
 
+    @property
+    def post_package_folder(self):
```

Comment:
Maybe the layout could be obviated? Users will be defined the current "post_package" folder as the package one in the "post_package()" method?